AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -549,6 +552,7 @@ vprok.ru#$#.fo-mobile-app-popup { display: none!important; }
 vprok.ru#$#body { overflow: visible!important; }
 diariosanrafael.com.ar##.toppersecond
 mashina.kg##.download-app-plashka
+coolors.co###sticky-cad
```

Comment:
`coolors.co###sticky-cad` is already in Easylist/Base

Reply:
removed